src/REVLoans.sol

```diff
@@ -103,7 +105,19 @@ contract REVLoans is ERC721, ERC2771Context, IREVLoans, Ownable {
     /// @notice The permit2 utility.
     IPermit2 public immutable override PERMIT2;
 
+    /// @notice The controller of revnets that use this loans contract.
+    IJBController public immutable override CONTROLLER;
+
     /// @notice Mints ERC-721s that represent project ownership and transfers.
+    IREVDeployer public immutable override DEPLOYER;
```

Comment:
wdyt about using the name `REVNETS` instead of `DEPLOYER`, the line `REVNETS.unrealizedAutoIssuanceAmountOf(revnetId)` feels a lot clearer in what its doing than if its named `DEPLOYER`

Reply:
love it. will change